modules/database/main.tf

```diff
@@ -4,14 +4,30 @@ data "aws_vpc" "vpc" {
   }
 }
 
+data "aws_kms_key" "vpc_key" {
+  key_id = "alias/${var.vpc_name}_vpc"
```

Comment:
Dette synes jeg ser rart ut. Kan du forklare kjapt hva dette her er, for nysgjerrighetens skyld? 😄

Reply:
Denne gir modulen tilgang til en KMS nøkkel for å kryptere/dekryptere verdier i Parameter store. Nøkler får en UUID, så derfor kan man også gi de et alias så det er lettere å se hva formålet med de er.